### PR TITLE
Assorted improvements to documentation

### PR DESCRIPTION
**Pull request by seabass-labrax**
## Improvements to `DOCS/new-license-workflow.md`

- Remove references to Travis CI now that the License List uses GitHub Actions for CI/CD

- Fix spelling and formatting

- Correct dead link to the SPDX Online Tools

### CODE REVIEW

**Review by swinslow — APPROVED** [review]
LGTM, thanks @seabass-labrax!

File: DOCS/new-license-workflow.md [hunk 1]
```diff
@@ -4,42 +4,41 @@
...
-3. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on [Overview page](https://spdx.dev/license-list/) for assistance.
-Ask the submitter for any additional info needed, preferably via the Github issue, if possible. Record any updates there.
+    3. If additional markup can accommodate the license, then the license does not need to be added: inform the requester, comment on the issue as such, then create a PR for the existing license with the additional markup, and close issue once the PR has been merged.
+3. If the submitter is not the license author or steward, ask for that contact or try to find that person or organization to make them aware the license has been submitted.
+4. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on [Overview page](https://spdx.dev/license-list/) for assistance.
+Ask the submitter for any additional info needed, preferably via the GitHub issue, if possible. Record any updates there.
```
Comment (jlovejoy) [inline DOCS/new-license-workflow.md:20]: just noticed, that is an outdated link, it should be "Explanation of SPDX License List Fields" with the link https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md

**Comment by jlovejoy** [comment]
LGTM, but one outdated link I noticed.
@seabass-labrax - can you update that? I think that's easier than me trying to do a PR against this one!

File: DOCS/new-license-workflow.md (hunk 1 above)
Comment (seabass-labrax) [reply DOCS/new-license-workflow.md:20]: There we go, @jlovejoy! :) Added commit 08a097cdf6e5b854c574d7b3022b0d9a01f69458.

**Pull request merged**